scripts/build/rollup.web.config.js

```diff
@@ -0,0 +1,27 @@
+import path from "path";
+import baseConfig from "./rollup.base.config.js";
+import resolve from "rollup-plugin-node-resolve";
+import commonjs from "rollup-plugin-commonjs";
+import globals from "rollup-plugin-node-globals";
+import json from "rollup-plugin-json";
+import alias from "rollup-plugin-alias";
+
+export default Object.assign(baseConfig, {
+  entry: "index.js",
+  dest: "prettier.js",
+  format: "umd",
+  plugins: [
+    alias({
```

Comment:
Stalled for the moment on this: the `alias` plugin has the following observed behavior:

- The code from `load-plugins-browser.js` is included
- The dependencies from `load-plugins.js` are also included

🤯 I have no idea. Will return to it with fresh eyes in the near future, or appreciate any help cracking the code in the meantime.